api/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.2 on 2023-05-01 12:07
+# Generated by Django 4.1.7 on 2023-05-13 19:26
```

Comment:
Huh? :) `poetry install`? :)
Not that it really matters, of course 😆

Reply:
Fixed it back to 4.2 and updated my local version :)